docs/entities/servicecatalog.md

```diff
@@ -0,0 +1,31 @@
+# ServiceCatalog
+
+A Service Catalog represents a group of Registered Services which are available in a Cluster Environment.
+
+## Specification
+
+The definition of a ServiceCatalog can be obtained directly from our [ServiceCatalog
+CRD](../../config/crd/bases/primaza.io_servicecatalogs.yaml). The specification
+contains a list of `ServiceCatalogService`. The Service Catalog Service contains the following fields:
```

Comment:
For a better review experience please format the markdown as follow. Oneline for each sentence, one empty line for creating a new paragraph.

```suggestion
The definition of a ServiceCatalog can be obtained directly from our [ServiceCatalog CRD](../../config/crd/bases/primaza.io_servicecatalogs.yaml).
The specification contains a list of `ServiceCatalogService`.
The Service Catalog Service contains the following fields:
```